core/blockchain.go

```diff
@@ -143,6 +143,55 @@ var defaultCacheConfig = &CacheConfig{
 	SnapshotWait:   true,
 }
 
+type BlockChainConfig struct {
```

Comment:
Exported structs like this should have comments explaining them

Reply:
Thanks for the suggestion I've added comment lines to explain `core.BlockChainConfig` struct.